catroid/src/main/java/org/catrobat/catroid/utils/IconsUtil.java

```diff
@@ -102,6 +106,25 @@ public static void mapIconSizesToDeviceSize() {
 		largeIconBottomBar = (int) (metrics.ydpi * (LARGE_ICON_SIZE_BOTTOM_BAR_IN_CM * INCH_TO_CM));
 	}
 
+	public static void addIconToLabel(ViewGroup parent, Brick brick, Sprite sprite, Context context) {
+		for (int i = 0; i < parent.getChildCount(); i++) {
+			View child = parent.getChildAt(i);
+			if (child instanceof LinearLayout || child instanceof RelativeLayout || child instanceof BrickLayout) {
```

Comment:
You could simply check for ViewGroup here (see my comment on #2485 )

Reply:
Fixed².